runtime/makelib/targets.mk.ftl

```diff
@@ -125,6 +125,11 @@ all: $(TARGETS)
 </#if>
 <#include "targets.mk.vendor.inc.ftl">
 
+# Add runtime include path for resource compiler on Windows.
+<#if uma.spec.type.windows>
+UMA_INCLUDES+=$(UMA_PATH_TO_ROOT)include
```

Comment:
This is a scary change, as it changes the include path for everything. Its possible it could introduce weird side affects. I expect most things already #include the include directory, but something may not. It also seems like a hack, as modules that need the include directory should specify that. Can the problem be fixed in a more specific way?

Reply:
Agreed and already updated the option specific to RC on Windows.